scripts/daily_plot.py

```diff
@@ -43,7 +45,11 @@ def show_values_on_bars(ax, label):
         # Species Count Total
         value = '{:n}'.format(p.get_width())
         bbox = {'facecolor': 'lightgrey', 'edgecolor': 'none', 'pad': 1.0}
-        ax.text(x, y, value, bbox=bbox, ha='center', va='center', size=9, color='darkgreen')
+        match conf['COLOR_SCHEME']:
```

Comment:
Match was only introduced with Python 3.10; this was giving the syntax error earlier. Could you change to a if/else?

Reply:
Correct, which is why I moved the version to 3.11. 

If we're not updating the Python version I can change this to an if/else - it's just a less elegant solution.